### DISCUSSION TITLE
New to BW - What's the best workflow when (a lot of) books have to be manually added ?

### DISCUSSION THREAD

**Discussion by ratoune2019** [bookwyrm-social/bookwyrm#3483] [Q&A]
Hi, 
I am new to BW, discovered thanks to a toot on Mastodon. Thanks so much for this wonderful tool :-) 

Context : 
This is the first time I participate on "Social Network Books" (no Goodreads account, nor any other account). But I have all my library registered in Zotero, which is so good to automatically collect information in databases such as Worldcat, Google books, and so easy to edit too, with all the different kind of data which are important for a book (included translator, number of edition, editors, url, and so on), thanks to its web browser connector.  I use also the plugin BookSearch in Obsidian to automatically populate my personal book database (Obsidian) for some books.
I want to add some books to my BW feed. 
Quite a lot of books I want to add do not exist in Inventaire, nor other instances. They exist in databases such as Worldcat. 

Options : 
1. I can add manually the books. Problem : it is terribly long to manually populate all fields for a single book AND it may create errors (typos...). 
2. I can create an account on Inventaire. Problem : I have to enter manually ISBN for any book I want to add (no camera to use the scanner feature). Furthermore, on Inventaire, I was not able to edit some data (a missing author for a book). 

In such a context, what would be the best solution for me ?
a. To populate my library in Inventaire first (or other app if better) ? What about the reliability of datas if it is not possible to edit an item there ?
b. To populate my library in BW first ? In such a case, is there a way to import datas (csv) from Zotero or Obsidian ?

Thanks for any suggestion !

**Comment by HelmutsCab** [comment]
Maybe this issue is interesting, it's  a feature request about importing book lists from Zotero. https://github.com/bookwyrm-social/bookwyrm/issues/3693

**Comment by hughrun** [comment]
There's also a CSV export from Zotero. That doesn't help right now but I believe someone's working on a "generic" CSV import where you would be able to choose columns to match to certain metadata (title, ISBN etc) so as a one-off import that would be a future option.
